### PR TITLE
implement joining chats, move icon-btn styles, rework dialogs

### LINKED ISSUE #31: Implement joining to chats
Join using chatId

### PR DESCRIPTION
**Pull request by maciejors**
closes #31

### CODE REVIEW

File: src/components/MemberListElement.svelte [hunk 1]
```diff
@@ -13,14 +13,17 @@
 
 	async function handlePromote() {
 		await promoteChatMember(chatId, memberId);
+		return true;
```
Comment (andrzejczukm) [inline src/components/MemberListElement.svelte]: Are these needed?

**Review by andrzejczukm — CHANGES_REQUESTED** [review]
What are those true values in return?

File: src/components/MemberListElement.svelte (hunk 1 above)
Comment (maciejors) [reply src/components/MemberListElement.svelte]: no, they are redundant, I tested something and forgot to remove that

**Review by andrzejczukm — APPROVED** [review]
great work

**Pull request merged**